### ISSUE TITLE
Multiple file uploads are not working in the subject set upload File Select dialog box.

### ISSUE THREAD

**Issue by twhyntie** [zooniverse/Panoptes-Front-End#280]
I can see from [here](https://github.com/zooniverse/Panoptes-Front-End/blob/4734da9c66216544b8da3d47fa5d9abb1e8ce19e/app/components/upload-drop-target.cjsx) that the `upload-drop-target` box has the `multiple` option in the `input` tag:

```
  render: ->
    <label className="upload-drop-target" data-active={@state.active ? null} style={position: 'relative'} {...@eventsToMakeDropWork()} onDrop={@handleDrop}>
      <input type="file" accept={@props.accept} multiple={@props.multiple} onChange={@handleFileSelection} style={HIDDEN_INPUT_STYLE} />
      {@props.children}
    </label>
```

However, I can't get multiple file selections working in Linux/Windows, Firefox/Chrome with the File Selection dialog box.

Looking at, e.g. [here](http://www.w3schools.com/tags/att_input_multiple.asp), does the `multiple` option need to be something like:

```
    <label className="upload-drop-target" data-active={@state.active ? null} style={position: 'relative'} {...@eventsToMakeDropWork()} onDrop={@handleDrop}>
      <input type="file" accept={@props.accept} multiple onChange={@handleFileSelection} style={HIDDEN_INPUT_STYLE} />
      {@props.children}
    </label>
```

i.e. not `multiple=[SOMETHING]`?

**Comment by brian-c** [comment]
`multiple` is a boolean attribute, `multiple="true"` is the same as just `multiple`. Will look into this further.

**Comment by brian-c** [comment]
Default was `false`, and I never overwrote it. Absolutely my bad.

**Comment by twhyntie** [comment]
Brilliant - thanks - this will make life infinitely easier :-)

Also closes [#265](https://github.com/zooniverse/Panoptes-Front-End/issues/265) for me (for first release).
